content/community/changelog/app-nuget/v5/breaking-changes/_index.nb.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Breaking changes
+description: Overview of breaking changes introduced into app nuget packages in v5.0.0.
+---
```

Comment:
Consider adding translate-to-norwegian tag as other pages not translated has this tag
```suggestion
tags: [translate-to-norwegian]
---
```